src/database/FirebaseArray.js

```diff
@@ -88,6 +88,13 @@
 
         this._sync.init(this.$list);
 
+        // $resolved provides quick access to the current state of the $loaded() promise.
+        // This information is useful in data-binding for knowing when the data is available.
```

Comment:
nitpick: this line is superfluous and adds no value (is actually a bit confusing)